### PR TITLE
details of ria

### PR DESCRIPTION
**Pull request by riaaniru2613**
esolves Issue #<1>

### Description
details of myself in introduction folder

### CODE REVIEW

**Comment by riaaniru2613** [comment]
@NJnisarg  please review my code and merge with the file

**Comment by NJnisarg** [comment]
@riaaniru2613 This PR you have created references issue #1 . The file you have commited is for issue #29 . Please make a seperate PR for #29  and keep this PR for #1 .
For now just put the ria.txt file back and remove the c file, for this PR

**Comment by riaaniru2613** [comment]
Is it fine now ?

**Comment by SaurabhAgarwala** [comment]
@riaaniru2613 for this PR you have to keep the _ria.txt_ file and remove the _fexpo.c_ file. For the _fexpo.c_ file create another PR when this one gets merged.

File: algorithms/29/fexpo.c [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        p = (p*p) % k;   
+    } 
+    return r; 
+} 
+ 
+int main() 
+{ 
+   int x,y,p;
+   printf("enter value of a,b and m : \n");
+   scanf("%u%u%u",&x,&y,&p;
```
Comment (yogesh1905) [inline algorithms/29/fexpo.c]: You missed a parenthesis after scanf

**Comment by riaaniru2613** [comment]
@yogesh1905 so i have to add another commit ? could you please tell me how to do it ?

**Comment by SaurabhAgarwala** [comment]
@riaaniru2613 I think this PR has become quite messy, which is what is confusing you. Suggest you to close this PR and make another fresh PR solving one issue at a time.

**Pull request closed without merging**